applications/rag/frontend/main.tf

```diff
@@ -58,6 +57,9 @@ resource "kubernetes_service" "rag_frontend_service" {
   metadata {
     name      = "rag-frontend"
     namespace = var.namespace
+    annotations = var.add_auth ? {
+      "beta.cloud.google.com/backend-config" = "{\"default\": \"${var.k8s_backend_config_name}\"}"
```

Comment:
is this a bug fix? does it work without this?

Reply:
this is replacement of kubernetes_annotation block in iap modules..

Reply:
No, please move back to iap module, we need to onboard ray too.

Reply:
ray already has from helm charts..
using kuberntes_annotations brings lot of diffs in terraform state.. after every apply there will be always some change around annotations..

Reply:
Ok, then we should add annotation `
      "cloud.google.com/neg" = "{\"ingress\": true}"`
      
    this annotation will be auto added first time apply, did you try re-apply?